src/kimmdy/assets/analyse_kimmdy_migration.py

```diff
@@ -0,0 +1,113 @@
+import MDAnalysis as mda
```

Comment:
maybe `assets` should be a `scripts` folder (or scripts is a subdirectory of assets)